docs/configuration/readme.md

```diff
@@ -1034,9 +1033,31 @@ See [ProxyPreserveHost](http://httpd.apache.org/docs/2.0/mod/mod_proxy.html#prox
 - Type: [base64 encoded] `string`
 - Optional
 
-Signing key is the base64 encoded key used to sign outbound requests. For more information see the [signed headers] docs.
+Signing Key is the key used to sign a user's attestation JWT which can be consumed by upstream applications to pass along identifying user information like username, id, and groups.
+
+If set, the signing key's public key will can retrieved by hitting Pomerium's `/.well-known/pomerium/jwks.json` endpoint which lives on the authenticate service. For example:
+
+```bash
+$ curl https://authenticate.int.example.com/.well-known/pomerium/jwks.json | jq
+```
+
+```json
+{
+  "keys": [
+    {
+      "use": "sig",
+      "kty": "EC",
+      "kid": "ccc5bc9d835ff3c8f7075ed4a7510159cf440fd7bf7b517b5caeb1fa419ee6a1",
+      "crv": "P-256",
+      "alg": "ES256",
+      "x": "QCN7adG2AmIK3UdHJvVJkldsUc6XeBRz83Z4rXX8Va4",
+      "y": "PI95b-ary66nrvA55TpaiWADq8b3O1CYIbvjqIHpXCY"
+    }
+  ]
+}
+```
 
-If no certificate is specified, one will be generated for you and the base64'd public key will be added to the logs.
+If no certificate is specified, one will be generated and the base64'd public key will be added to the logs. Note, hoever, that this key be unique to each service, ephemerial, and will not be accessible via the authenticate service's `jwks_uri` endpoint.
```

Comment:
typo: `hoever` should be `however`, `ephemerial` should be `ephemeral`

Reply:
@calebdoxsey I'm having one of those days...